stackrox/base/services/stackrox-services/central-service.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# Source: stackrox-central-services/templates/01-central-14-service.yaml
+apiVersion: v1
+kind: Service
```

Comment:
can you move these out of the `stackrox-services` directory, just so it's consistent with the rest of the structure for the other resources.